.github/workflows/_build_base.yaml

```diff
@@ -57,7 +57,27 @@ permissions:
 
 jobs:
 
+  nvidia-require-cuda:
+    runs-on: ubuntu-22.04
+    outputs:
+      NVIDIA_REQUIRE_CUDA: ${{ steps.metadata.outputs.NVIDIA_REQUIRE_CUDA }}
+    steps:
+      - name: Check out the repository under ${GITHUB_WORKSPACE}
+        uses: actions/checkout@v4
+
+      - name: Get NVIDIA_REQUIRE_CUDA env variable from AMD64 CUDA image
+        id: metadata
+        shell: bash -x -e {0}
+        run: |
+          BASE_IMAGE=${{ inputs.BASE_IMAGE }}
+          if [[ $BASE_IMAGE == latest ]]; then
+            BASE_IMAGE=$(cat .github/container/Dockerfile.base | sed -n "s/^ARG BASE_IMAGE=\(.*\)$/\1/p")
+          fi
+          NVIDIA_REQUIRE_CUDA=$(docker run $BASE_IMAGE bash -ec "echo \$NVIDIA_REQUIRE_CUDA" | tail -1)
```

Comment:
Not sure it's worth it here, but this requires fetching the whole image and starting it, which is not needed.
Something like
```bash
skopeo inspect "docker://${BASE_IMAGE}" | jq -r '.Env[] | select(startswith("NVIDIA_REQUIRE_CUDA=")) | ltrimstr("NVIDIA_REQUIRE_CUDA=")'
```
should work and only requires fetching the manifest 🤷

Reply:
Make sense. I will give a shot. Did not know about `skopeo` function